server/controllers/finance/reports/financial.patient.handlebars

```diff
@@ -4,9 +4,20 @@
 
   {{> header }}
 
-  <h3 class="text-center text-uppercase"><strong>{{translate "PATIENT_RECORDS.FINANCIAL_ACTIVITY.REPORT_TITLE"}}</strong></h3>
-  <h4 class="text-center"><strong>{{ patient.reference }} | {{ patient.display_name }}</strong></h4>
-  <h4 class="text-center"><strong>{{ patient.debtor_group_name }} ({{ patient.number }})</strong></h4>
+  <h3 class="text-center text-uppercase">
+    <span style="text-decoration-line: underline">
+      <strong> {{translate "PATIENT_RECORDS.FINANCIAL_ACTIVITY.REPORT_TITLE"}} </strong>
+    </span>  
+  </h3>
+  <h4 class="text-uppercase"><strong>{{ patient.reference }} | {{ patient.display_name }}</strong></h4>
+  <h4 class="text-uppercase"><strong>{{ patient.debtor_group_name }} ({{ patient.number }})</strong></h4>
+
+  <h4 class="text-uppercase" style="background-color: #efefef;">
+    <strong>
```

Comment:
Why can't we just look at the bottom of the report for this...?  Repeating it twice doesn't make that much sense to me. :/